content/en/blog/2020/docker-rate-limit/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Handling Docker Hub rate limiting
+description: How to ensure your clusters are not impacted by Docker Hub rate limiting.
+publishdate: 2020-12-05
+attribution: John Howard (Google)
+keywords: [docker]
+target_release: 1.8
+---
+
+Since November 20th, 2020, Docker Hub has introduced [rate limits](https://www.docker.com/increase-rate-limits) on image pulls.
+
+Because Istio uses [Docker Hub](https://hub.docker.com/u/istio) as the default registry, usage on a large cluster may lead
+to pods failing to startup due to exceeding rate limits. This can be especially problematic for Istio, as there is typically
+the Istio sidecar image alongside most pods in the cluster.
+
+# Mitigations
+
+Istio allows using a custom docker registry. This can be configured by passing `--set hub=<some-custom-registry>` at installation time.
+
+Istio provides official mirrors to [Google Container Registry](`gcr.io/istio-release`). This can be configured with `--set hub=gcr.io/istio-release`. This is available for Istio 1.5+.
+
+Alternatively, you can copy the official Istio images to your own registry. This is especially useful if your cluster runs in an environment with a registry tailored for your environment (for example, on AWS you may want to mirror images to Amazon ECR). This can be done with the following script:
```

Comment:
```suggestion
Alternatively, you can copy the official Istio images to your own registry. This is especially useful if your cluster runs in an environment with a registry tailored for your use case (for example, on AWS you may want to mirror images to Amazon ECR) or you have air-gapped security requirements where access to public registries is restricted. This can be done with the following script:
```